docker/Dockerfile

```diff
@@ -23,6 +25,8 @@ RUN pacman -Syyuu --noconfirm --needed --overwrite \*
 
 # Add a script that installs packages
 COPY usr/local/bin/install-package /usr/local/bin/install-package
+COPY etc/pacman/pacman.latest.conf /etc/pacman.latest.conf
+COPY etc/pacman/mirrorlist.latest /etc/pacman.d/mirrorlist.latest
```

Comment:
What are these ones doing

Reply:
Copying files. They are internal dependencies of the install-package script. Note that I've been to amalgamate the mirror list file into the other config file.